tt_metal/impl/allocator/allocator.cpp

```diff
@@ -378,14 +378,14 @@ void verify_safe_allocation(Allocator& allocator) {
     }
 }
 
-uint64_t allocate_buffer(
-    Allocator &allocator,
-    DeviceAddr size,
-    DeviceAddr page_size,
-    const BufferType &buffer_type,
-    bool bottom_up,
-    std::optional<uint32_t> num_shards) {
+const std::set<Buffer *> &get_allocated_buffers(const Allocator &allocator) { return allocator.allocated_buffers; }
```

Comment:
why allocated buffers is set but not unordered_set

Reply:
I will change to unordered_set